allure-report-plugin/src/main/kotlin/io/qameta/allure/gradle/report/tasks/AllureReport.kt

```diff
@@ -36,9 +43,10 @@ open class AllureReport @Inject constructor(objects: ObjectFactory) : AllureExec
 
     @TaskAction
     fun generateAllureReport() {
+        val injected = project.objects.newInstance<Injected>()
```

Comment:
access to `project` in task action is incompatible with the configuration cache.

Reply:
yep, the idea was to get rig of `project` usages but all in all I returned to project usage 🤦